rskj-core/src/main/java/co/rsk/peg/union/UnionBridgeStorageProvider.java

```diff
@@ -10,5 +10,7 @@ public interface UnionBridgeStorageProvider {
     Optional<RskAddress> getAddress(ActivationConfig.ForBlock activations);
     void setLockingCap(Coin lockingCap);
     Optional<Coin> getLockingCap(ActivationConfig.ForBlock activations);
+    Optional<co.rsk.core.Coin> getWeisTransferredToUnionBridge(ActivationConfig.ForBlock activations);
+    void setWeisTransferredToUnionBridge(co.rsk.core.Coin weisTransferred);
```

Comment:
Idea: if instead of a setter method we have 2 separate methods `increase` and `decrease`? Feel like that is less error prone and gives us control on how we manipulate this value. A setter method let's set any value you want any time

Reply:
It looks good to me to have some constraints in the storage layer to reduce the risk of making a mistake with a critical amount of balance. This implies a refactor that I would like to add once we have the `no-rskip-check` refactor done. Wdyt?